multi-model-service/docs/code/access.md

```diff
@@ -0,0 +1,271 @@
+# Access & Usage Information
+
+## Basic Information
+
+### Model Server URL
+
+The model server can be accessed [here](https://model-service.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/).
+
+## Authentication
+
+In order to gain access to the model server you will need to sign in with your SSO credentials. Once you have signed in you are able to generate a token by navigating to **`Apps and API Keys`** on the top ribbon and hitting **`Create new Application`**:
+
+![Generation Example](../images/generation-example.png)
+
+**Note:** If you are unable to access the model server via SSO you need to reach out to a platform administrator.
+
+## API Schema
+<!--
+The name of the api, model-service-api, is grabbed from the name field in the ai-catalog.yaml metadata for the api.
+
+TechDocs read this as a URL starting from where the docs are located, so we can use ../ behaviour to navigate the TechDocs to reference other resources/components/apis
+-->
+The API Schema is available [here](../../../../../api/model-service-api/definition).
+
+## Usage Examples
+<!--
+Sourced from: https://model-service.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/examples
+-->
+
+### IBM Granite-8B-Code-Instruct
+
+#### Text Generation
+
+##### Using Curl
+
+```
+curl -X 'POST' \
+    'https://ibm-granite-8b-code-instruct-3scale-apicast-production.<hostname>:<port>/v1/completions' \
```

Comment:
```suggestion
    'https://ibm-granite-8b-code-instruct-3scale-apicast-production.apps.rosa.redhat-ai-dev.m6no.p3.openshiftapps.com/v1/completions' \
```

Reply:
WDYT? As we provide a specific access point above I think is ok to have a specific url here too.

Reply:
I had a conversation with @johnmcollier and we decided to try to limit the references to the url so I blanked those out. The URL was already in git but we didn't want to spam it everywhere

Reply:
Hmm, that's a good point and I think we could also invest time to see how/if we could remove it completely.

Reply:
Yeah it wouldn't be possible to remove it since it was there initially but it is protected behind the SSO and stuff so that isn't a concern, I believe our thought process was to reduce the amount of times it shows up to _try_ and limit the traffic anyway